molgenis-core-ui/src/main/resources/css/molgenis.css

```diff
@@ -174,4 +174,29 @@ body {
 	width: 125px;
 	padding: 7px 0px;
 	height: 26px;
-}
\ No newline at end of file
+}
+
+/* Change threshold for collapsing the menu, default is 768px*/
+@media (max-width: 768px) {
```

Comment:
Let's make this 1127, that should fix the issue for the standard case, even if the language pulldown is shown.